package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.34",
+  "publisher": "Aqovia",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
```

Comment:
Please review the [extensions suggested by the NE dev team](https://aqovia.atlassian.net/wiki/spaces/~288291211/pages/2633269251/VSCode+-+making+it+a+nicer+place+to+be) for possible inclusion. Some point to note:
- I would not include Tabnine (they are using the free version as a limited alternative to Github Copilot which we need to sort out their licensing for)
- Similarly look out for anything else that's a potential duplicate. If in doubt, leave it out - we can and probably should, invite teams to submit PRs after the initial version.

Reply:
Added - 
.NET Core Test Explorer
Auto-Using for C#
C# Interpolated String Converter
GitLens
Import Cost VSCode Extension
Inline YAML Syntax Highlighting
Markdown Preview Enhanced
Material Icon Theme
PDF Preview
Python Test Explorer for Visual Studio Code
string-highlight
Test Explorer UI